css/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+}
+
+.head {
+  display: flex;
+  justify-content: center;
+}
+
+h1 {
+  border: 2px solid black;
+  border-radius: 10px;
+  padding: 5px 100px;
+}
+
+.card-body {
+  border: 2px solid red;
+  height: 700px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 2px solid blue;
+  margin: 15px;
+  margin-top: 25px;
+  width: 300px;
+  height: 200px;
+}
+
+.question {
+}
+
+[type="button"] {
```

Comment:
Why do you use this selector instead of the class `.answer`? Besides, the class should be better called `answer-button` or `button-answer`.